src/DataService/Access/Account.php

```diff
@@ -11,8 +11,7 @@
  *   service="Access Data Service",
  *   baseUri="https://access.auth.theplatform.com/data/Account",
  *   schemaVersion="1.0",
- *   path="/data/Account",
- *   hasAccountContext=false
```

Comment:
I read in their docs that _all_ data services support account contexts, so there's no need for it in the annotation.